app/models/partner_request.rb

```diff
@@ -1,5 +1,5 @@
 class PartnerRequest < ApplicationRecord
-  belongs_to :partner, dependent: :destroy
+  belongs_to :partner
```

Comment:
See [SO](https://stackoverflow.com/a/34625321/3557284).  My understanding is you only need it on the `has_many` side; otherwise, you can end up with orphaned records, depending on how things are done.  I'm happy to put it back if it's really needed and won't break things.

Reply:
Once I read it closer I realized you're right! Thanks for this!